endpoints/openrtb2/video_auction_test.go

```diff
@@ -484,6 +484,58 @@ func TestVideoEndpointValidationsSiteMissingRequiredField(t *testing.T) {
 	assert.Len(t, podErrors, 0, "Pod errors should be empty")
 }
 
+func TestVideoEndpointValidationsMissingVideo(t *testing.T) {
+	ex := &mockExchangeVideo{}
+	deps := mockDeps(t, ex)
+	deps.cfg.VideoStoredRequestRequired = true
+
+	durationRange := make([]int, 0)
+	durationRange = append(durationRange, 15)
+	durationRange = append(durationRange, 30)
+
+	pods := make([]openrtb_ext.Pod, 0)
```

Comment:
`pod1` and `pod2` are not used beyond creation of the slice. Consider replacing lines 496-508 with a literal initialization.

.. and actually, same goes for `durationRange`, `mimes`, and `videoProtocols`.

Reply:
Updated to be literal initializations. Removed `mimes` and `videoProtocols` as those weren't needed either.